core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -1129,7 +1129,7 @@ package object config {
   private[spark] val LOCALITY_WAIT_RACK = ConfigBuilder("spark.locality.wait.rack")
     .fallbackConf(LOCALITY_WAIT)
 
-    private[spark] val REDUCER_MAX_SIZE_IN_FLIGHT = ConfigBuilder("spark.reducer.maxSizeInFlight")
+  private[spark] val REDUCER_MAX_SIZE_IN_FLIGHT = ConfigBuilder("spark.reducer.maxSizeInFlight")
```

Comment:
You're not changing anything else in this file, so better not touch this since it's unrelated.

Reply:
ok